src/main/java/corgiaoc/byg/config/biomeweight/ConfigWeightManager.java

```diff
@@ -43,6 +43,7 @@
     public static final ConfigWeight GUIANA_SHIELD = new ConfigWeight("Guiana_Shield", 5);
     public static final ConfigWeight GROVE = new ConfigWeight("Grove", 5);
     public static final ConfigWeight ENCHANTED_GROVE = new ConfigWeight("Enchanted_Grove", 1);
+    public static final ConfigWeight ICEBURG_OCEAN = new ConfigWeight("Iceburg_Ocean", 4);
```

Comment:
Should be spelled "iceberg". Do not use this class anymore and utilize the overrides provided to you from the BYGBiome class in the latest commits on this branch.